### DISCUSSION TITLE
How to reset videojs ControlBar on error?

### DISCUSSION THREAD

**Discussion by OutofSkills** [videojs/video.js#8473] [Q&A]
Is there a way to reset the player's previous video duration, and current time when an error is triggered. I'm struggling with this small issue for a while and I wasn't able to find a good solution, other than css:). Also I tried the player.currentTime(0) and player.duration(0) methods but it ends with an issue on iOS, it's setting a vjs-ended class and displaying the replay button when having an error and then playing another video.

**Comment by amtins** [comment]
@OutofSkills what about `player.reset()` ?

Something like:
```javascript
player.on('error', ()=>player.reset())
```
